Tests/PluginGeneratorTests/JarChecker.cs

```diff
@@ -27,7 +28,11 @@ public void JarContainsFiles(params string[] expectedRelativePaths)
         {
             foreach (string relativePath in expectedRelativePaths)
             {
-                string fullFilePath = Path.Combine(this.unzippedDir, relativePath);
+                string[] matchingFiles = Directory.GetFiles(this.unzippedDir, relativePath, SearchOption.TopDirectoryOnly);
+
+                Assert.IsTrue(matchingFiles.Length < 2, "Test error: supplied relative path should not match multiple files");
+                string fullFilePath = matchingFiles.FirstOrDefault();
+
```

Comment:
The generated XML files are now uniquely named so we have to located them using wildcards. This change checks that only one file matches the wildcard expression.